private-undistributed.yml

```diff
@@ -314,6 +314,7 @@ allow-licenses:
   - 'BSD-3-Clause AND BSD-3-Clause-Clear'
   - 'BSD-2-Clause AND BSD-3-Clause AND Python-2.0'
   - 'BSD-3-Clause AND LicenseRef-scancode-protobuf'
+  - 'BSD-3-Clause AND LicenseRef-scancode-unknown-license-reference AND MIT'
```

Comment:
This is detected as an unknown license and probably shouldn't be added to the allowedlist.
I'll let legal get the final say